extractor2/README.md

```diff
@@ -0,0 +1,23 @@
+# Berserker Extractor2
```

Comment:
You are right. 
I was planing to simplify the projects structure after #22 merged, and then remove the old `./{extractor,normalizer}` in next PR to avoid growing this one even further.